_assets/javascripts/main.js

```diff
@@ -9,4 +9,7 @@
 //= require nav-var.js
 //= require about-text.js
 //= require scrolldiv.js
+//= require bootstrap.min.js
```

Comment:
Is ` //= require bootstrap.min.js` necessary? We already have `//= require vendor/bootstrap.js` above

Reply:
solved ;)